Library/Formula/emp.rb

```diff
@@ -0,0 +1,40 @@
+require "language/go"
+
+class Emp < Formula
+  desc "CLI for Empire."
+  homepage "https://github.com/remind101/emp"
+  url "https://github.com/remind101/emp/archive/v0.9.1.tar.gz"
+  sha256 "9619ffd25856988d92d39b31d9e7ed6edacbd85543ff6e60300954df7c16135d"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    emppath = buildpath/"src/github.com/remind101/emp"
+    emppath.install Dir["{*,.git}"]
```

Comment:
Would you mind giving me a spot on how the change would be exactly? I'm not too sure how the change changes the build process. I made the change and the formula fails to install.